File: app/src/main/java/io/nanodegree/andrea/popularmovies/model/MovieContainer.java
package io.nanodegree.andrea.popularmovies.model;

import com.squareup.moshi.Json;

import java.util.List;

/**
 * Created by Andrea Loddo (andrea@evenly.io) on 20.12.2018
 * for Evenly GmbH
 * <p>
 * Copyright (c) 2018 Evenly GmbH,
 * all rights reserved
 */
public class MovieContainer {

    @Json(name = "results")
    public List<Movie> movieList;
}

File: app/src/main/java/io/nanodegree/andrea/popularmovies/data/retrofit/response/MovieContainer.kt
package io.nanodegree.andrea.popularmovies.data.retrofit.response

import com.google.gson.annotations.SerializedName

import io.nanodegree.andrea.popularmovies.data.model.Movie

data class MovieContainer (@field:SerializedName("results") val movieList: List<Movie>?)

